openhtmltopdf-examples/src/main/resources/freemarker/many_pages.ftl

```diff
@@ -0,0 +1,627 @@
+							CÓDIGO PRESUPUESTARIO
+						</th>
+					</tr>
+					<tr>
+						<th class="hdrAsig" style="border-top: 1px solid;border-left: 1px solid">
+							SECT
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							PROG
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							SUBP
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							PROY
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							ACTI
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							PART
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							GENE
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid">
+							ESPE
+						</th>
+						<th class="hdrAsig" style="border-top: 1px solid; border-right: 1px solid">
+							SUBE
+						</th>
+					</tr>
+					</thead>
+					<tbody>
+					<tr>
+
+						<td class="hdrAsig"
+							style="border-top: 1px solid; border-bottom: 1px solid; border-left:1px solid">
+							01
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							01
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							09
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							02
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							03
+						</td>
+
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							450
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							02
+						</td>
+						<td class="hdrAsig" style="border-top: 1px solid; border-bottom: 1px solid">
+							02
+						</td>
+						<td class="hdrAsig"
+							style="border-top: 1px solid; border-bottom: 1px solid; border-right: 1px solid">
+							01
+						</td>
+					</tr>
+					</tbody>
+				</table>
+			</th>
+		</tr>
+		<tr>
+			<th colspan="10"></th>
+		</tr>
+		<tr>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				ASIENTO
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				FECHA
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				DETALLE
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				ACTUALIZADO
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				COMPROMISO
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				AFECTADO POR
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				POR COMPROMETER
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				CAUSADO
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				AFECTADO POR
+			</th>
+			<th class="hdrAsig" style="border-bottom: 1px solid">
+				PAGADO
+			</th>
+		</tr>
+		</thead>
+
+		<tbody>
+			[#list 1..1000 as subRow]
```

Comment:
This count is high at the moment, before merging this branch it should be decreased to a sensible value, as travis won't finish whit such many rows...